src/Symfony/Component/AssetMapper/ImportMap/ImportMapEntry.php

```diff
@@ -39,12 +40,12 @@ private function __construct(
 
     public static function createLocal(string $importName, ImportMapType $importMapType, string $path, bool $isEntrypoint): self
     {
-        return new self($importName, $importMapType, $path, $isEntrypoint, null, null);
+        return new self($importName, $importMapType, $path, $isEntrypoint, null, null, null);
     }
 
-    public static function createRemote(string $importName, ImportMapType $importMapType, string $path, string $version, string $packageModuleSpecifier, bool $isEntrypoint): self
+    public static function createRemote(string $importName, ImportMapType $importMapType, string $path, string $version, string $packageModuleSpecifier, bool $isEntrypoint, ?string $resolver = null): self
     {
-        return new self($importName, $importMapType, $path, $isEntrypoint, $version, $packageModuleSpecifier);
+        return new self($importName, $importMapType, $path, $isEntrypoint, $resolver, $version, $packageModuleSpecifier);
```

Comment:
This would break BC no ?

Reply:
IMHO no, as a default resolver is provided in the asset mapper config
As this part is not exposed (I think), it will just be resolved internally